spidermon/contrib/validation/translator.py

```diff
@@ -2,6 +2,15 @@
 import re
 
 
+schematics_installed = False
+try:
+    import schematics
+
+    schematics_installed = True
+except ImportError:
+    pass
+
+
```

Comment:
This seems to be generic file for validations, can we not move schematics related logic inside schematics/translator ?

Reply:
You are right. I am changing it.